src/set_theory/cardinal/basic.lean

```diff
@@ -565,27 +565,13 @@ cardinal.wf.conditionally_complete_linear_order_with_bot 0 $ le_antisymm (cardin
 
 instance wo : @is_well_order cardinal.{u} (<) := ⟨cardinal.wf⟩
 
-/-- The set in the definition of `cardinal.succ` is nonempty. -/
+/-- The set in the definition of `order.succ` for cardinals is nonempty. -/
 theorem succ_nonempty (c : cardinal) : {c' : cardinal | c < c'}.nonempty := ⟨_, cantor c⟩
 
-/-- The successor cardinal - the smallest cardinal greater than
-  `c`. This is not the same as `c + 1` except in the case of finite `c`. -/
-def succ (c : cardinal) : cardinal :=
-Inf {c' | c < c'}
```

Comment:
You should add a lemma that says `succ c = Inf {c' | c < c'}` since `rw succ` won't work any more.

Reply:
I don't think this result is particularly interesting on its own, especially given its immediate corollaries. But it can't hurt.